contextualized/easy/wrappers/SKLearnWrapper.py

```
@@ -564,7 +565,9 @@ def fit(self, *args, **kwargs) -> None:
             "n_bootstraps", self.n_bootstraps
         )
         for bootstrap in range(self.n_bootstraps):
-            model = self.base_constructor(**organized_kwargs["model"])
+            model_args = dict(organized_kwargs["model"])
+            model_args.pop("encoder_kwargs", None)
```

This will fail silently if a user tries to use "encoder_kwargs". Now that lightning_modules use a flat arg structure, SKLearnWrapper needs to obey this.

---

Expanded